addons/website/static/src/js/menu/edit.js

```diff
@@ -185,6 +185,7 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
      * @param {OdooEvent} ev
      */
     _onEditionWillStop: function (ev) {
+        $('.dropdown_mega_menu').dropdown('hide');
```

Comment:
Feels like this should be done for all dropdown menu, directly by the editor

Reply:
Then is this really related to this task ? Just wondering

Reply:
Should be in a separate commit but as your task needs it... yes it is related to this task